optaplanner-build-parent/pom.xml

```diff
@@ -19,8 +19,8 @@
     <!-- Dependencies -->
     <!-- ************************************************************************ -->
 
-    <!-- SNAPSHOT dependency versions -->
-    <version.org.kie>7.40.0-SNAPSHOT</version.org.kie>
+    <!-- SNAPSHOT and KIE dependency versions -->
+    <version.org.kie.kogito>0.11.1</version.org.kie.kogito>
```

Comment:
0.12 is out already.

Reply:
Also, we should discuss how we become part of the Kogito train.
(Not for this PR though.)

Reply:
Upgraded to 0.12. Yes, we must become part of the kogito release train before September. Ideally, the fix their drools 7 dependency issue first.